appveyor.yml

```diff
@@ -10,14 +10,14 @@ install:
 build_script:
     - cmd: build.bat canary
 
-nuget:
-    disable_publish_on_pr: true
+#nuget:
+#    disable_publish_on_pr: true
 
 for:
 
     - branches:
           only:
-              - /^(master|\d+\.x)$/
+              - /^(master|\d+\.x|feature\/ci)$/
```

Comment:
This is temporary for us to validate the packages being pushed based on builds from this PR to https://ci.appveyor.com/nuget/ecs-dotnet